docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -95,5 +95,7 @@ Where to deploy the monitor. Monitors can be deployed in multiple locations so t
 Control the monitor's download speeds, upload speeds, and latency to simulate your application's behavior on slower or laggier networks.
 `screenshot` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/common_types.ts#L190[`ScreenshotOptions`])::
 Control whether or not to capture screenshots. Options include `'on'`, `'off'`, or `'only-on-failure'`.
+`privateLocations` (`Array<string>`)::
+Where to deploy the monitor when using <<private-locations,private locations>>. Monitors can be deployed in multiple locations including both `locations` and `privateLocations` so that you can detect differences in availability and response times across those locations.
```

Comment:
IMO it's not clear what's the difference between both, just that you may want to detect differences in available and response times.

Here's what I'd write:

```suggestion
The <<private-locations,private locations>> to which the monitors will be deployed. These private locations refer to locations hosted and managed by you, whereas <<locations,locations>> are hosted by Elastic.
```